Makefile

```diff
@@ -10,15 +10,59 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-all: csv-generator
+all: operator
 
-csv-generator:
-	CGO_ENABLED=0 go build -a -ldflags '-extldflags "-static"' -o build/_output/csv-generator tools/csv-generator.go
+csv-generator: build/_output/csv-generator
+build/_output/csv-generator: tools/csv-generator.go
+	CGO_ENABLED=0 go build -a -ldflags '-extldflags "-static"' -o $@ $<
+
+operator-image: build/_output/csv-generator build/_registry/image_label
+	operator-sdk build --image-builder buildah $$(cat build/_registry/image_label)
```

Comment:
I would be in favor of using https://quay.io/repository/openstack-k8s-operators/builder-toolbox to invoke commands such as operator-sdk or buildah.

This way we will have consistent set of tools and with the image tags we can pin or select specific version of those tools independent of host OS.